draft-ietf-dprive-xfr-over-tls.md

```diff
@@ -323,12 +321,22 @@ applicable to XFR-over-TLS as well.
 It is RECOMMENDED that clients and servers that support XoT also implement
 EDNS0 Keepalive [RFC7828].
 
+It is useful to note that in these mechanisms is it the secondary that initiates the TLS connection to the primary for a XFR request, so that in terms of connectivity the secondary is the TLS client and the primary the TLS server.
+
+## TLS versions
+
+For improved security all implementations of this specification MUST use only TLS 1.3 [RFC8446] or later.
```

Comment:
We could also have a reference to https://tools.ietf.org/html/rfc7525 in the document for TLS guidance

Reply:
We could but RFC7858 (DoT) and RFC8310 (DoT profiles) already say that DoT implementations must follow that guidance?